packages/fx-core/src/plugins/resource/bot/v2/mapping.ts

```diff
@@ -55,6 +56,15 @@ const projectFileMap = new Map<Runtime, (appName: string) => string>([
   [Runtime.Dotnet, (appName: string) => `${appName}.csproj`],
 ]);
 
+export const moduleMap: { [key: string]: string } = {
```

Comment:
Would it be easier to define moduleMap as a normal Map type for moduleMapCapitalized() method?

Reply:
If `moduleMap.get(key)` could be `undefined`. Since all key value pairs are static in this case, I prefer using object rather than map.